CMakeLists.txt

```diff
@@ -21,6 +21,11 @@ endif()
 # set custom module path for findFFmpeg
 set(CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake/modules")
 
+if(BUILD_CPP)
+	# we want windows libraries like FAudio.dll instead of libFAudio.dll
```

Comment:
This is actually true of all the Windows builds - maybe we should just set this to true for WIN32 and MINGW, just in case someone generates this for VS or something.